src/main/resources/hazelcast.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<hazelcast xmlns="http://www.hazelcast.com/schema/config"
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xsi:schemaLocation="http://www.hazelcast.com/schema/config
+           http://www.hazelcast.com/schema/config/hazelcast-config-4.0.xsd">
```

Comment:
I would use hazelcast.yaml config then we do not have to update xsd version all the time.

Reply:
Spring doesn't support auto bean initialization if we use `hazelcast.yaml`.

Reply:
Are you sure? I used it [here](https://github.com/hazelcast/hazelcast-code-samples/tree/master/hazelcast-integration/kubernetes/samples/external-client) and I'm pretty sure it works correctly.

Reply:
Ok, it wasn't supported in the version I first used. Changed it 👍